server/plugin/plugin.go

```diff
@@ -227,6 +252,9 @@ type GitHubUserInfo struct {
 	LastToDoPostAt      int64
 	Settings            *UserSettings
 	AllowedPrivateRepos bool
+
+	// MM34646ResetTokenDone is set for a user whose token has been reset for MM-34646.
+	MM34646ResetTokenDone bool
```

Comment:
I'm all for linking ticket in code comment for additional context. But naming a variable after it seems to much as it doesn't add clarity to the code. Can this be simply `ResetTokenDone`?

Reply:
1/5 I disagree. The names should be intention-revealing, and `ResetTokenDone` is not specific enough